database.sql

```diff
@@ -3,4 +3,9 @@ create table github_discord_url(
     created_at timestamptz DEFAULT Now() not null,
     updated_at timestamptz DEFAULT Now() not null,
     discord_url varchar(255) not null UNIQUE
+);
+
+create table auth_user_info(
```

Comment:
did this work in your local? I thought it might not, since this file already ran - dunno for sure in this framework